BaseStation/MainWindow/MapTileDownloadManager.cs

```diff
@@ -69,12 +73,33 @@ public static String Fetch(MapConfiguration config)
                 byte[] hash = sha.ComputeHash(buffer);
                 var bufferHash = BitConverter.ToString(hash).Replace("-", String.Empty).Substring(0, 16);
                 Console.WriteLine(bufferHash);
-                String fileName = Directory.GetCurrentDirectory().ToString()+ @"\Images\" + bufferHash + ".jpg";
+                String fileName = Directory.GetCurrentDirectory().ToString()+ @"\Images\" + bufferHash + ".png";
                 buffer.Position = 0;
                 using (var file = File.Create(fileName))
                 {
                     buffer.CopyTo(file);
-                    return bufferHash;
+                }
+                CropImage(fileName, config);
```

Comment:
Can we get the image to crop directly off of `buffer` instead of saving it to a file, then loading it and saving it again?